apps/web/components/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import Link from "next/link";
+
+export default function Footer() {
+  return (
+    <footer className="bg-zinc-800 py-12 lg:py-16">
+      <div className="container">
+        <div className="flex flex-col items-center lg:flex-row lg:items-center lg:justify-between">
+          <a
+            className="text-lg font-bold leading-none md:text-xl xl:text-2xl"
+            href="/"
+          >
+            Template Bhai
+          </a>
+          <nav className="mt-4 flex flex-wrap justify-center gap-x-8 gap-y-2 lg:mt-0">
+            <Link href="/templates">
```

Comment:
These are used for both menu and footer, it can be probably exported into a separate component